modules/common/src/main/scala/notebook/front/widgets/package.scala

```diff
@@ -11,705 +11,15 @@ import notebook.front.widgets.magic
 import notebook.front.widgets.magic._
 import notebook.front.widgets.magic.Implicits._
 import notebook.front.widgets.magic.SamplerImplicits._
-import widgets.Defaults.DEFAULT_MAX_POINTS
 
 /**
  * This package contains primitive widgets that can be used in the child environment.
  */
-package object widgets {
-  object Defaults {
-    val DEFAULT_MAX_POINTS = 1000
-  }
-  def scopedScript(content: String, data: JsValue = null, selector:Option[String]=None) = {
-    val tag = <script type="text/x-scoped-javascript">/*{xml.PCData("*/" + content + "/*")}*/</script>
-    val withData = if (data == null)
-      tag
-    else
-      tag % new UnprefixedAttribute("data-this", Json.stringify(data), Null)
+package object widgets
+  extends Generic
+  with Texts
+  with Lists
+  with Layouts
+  with charts.Charts {
```

Comment:
why do we'd need package object extending traits :) ?

`import widgets._` would work for all `classes`. it wouldn't work only for `def`s not inside package object.

but isn't that we wanted to have less things inside package object for serializing less?